Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -716,6 +724,12 @@ static Color activeButtonTextColor()
             return semanticColorFromNSColor(alternateColors[1]);
         }
 
+        // FIXME: Remove this fallback when AppKit without tertiary-fill is not used anymore; see rdar://108340604.
+        case CSSValueAppleSystemTertiaryFill:
+            if (localAppearance.usingDarkAppearance())
+                return { SRGBA<uint8_t> { 255, 255, 255, 13 }, Color::Flags::Semantic };
+            return { SRGBA<uint8_t> { 0, 0, 0, 13 }, Color::Flags::Semantic };
```

Comment:
Let's put this in an `#if !HAVE(NSCOLOR_FILL_COLOR_HIERARCHY)`.

Reply:
I was thinking that the fallback could *always* be there, because the css value doesn't have a build condition -- should it? (And is it possible? I didn't see any in CSSValueKeywords.in. 🤔

Reply:
But I'll add the `#if` anyway; let me know if I should remove it after all...

Reply:
The build condition in CSSValueKeywords.in, is `PLATFORM(MAC)`.

But you're right that we should probably always have the fallback without the `#if` until the staging code is removed (which can be done in a couple weeks), to keep tests passing. The `HAVE` is going to be around for longer than that.